contracts/token/Forta.sol

```diff
@@ -36,7 +40,7 @@ contract Forta is FortaCommon {
      * @return version of FORT deployed in L1
```

Comment:
We should have better documentation about what this version number is meant to communicate, and instructions for when and how it should be incremented. For example, should every upgrade have a new version? Only if the interface changes? Should it follow semver? Or what does each component of the version represent?

I'm not sure if bumping from 0.1.0 to 0.1.1 here respects the expectations that we've set for the function getter.

Reply:
The idea is to follow semver. Fair enough, removing whitelist should be a minor, not a patch.
I'm updating the README with more strict definitions.
I'm noticing also the global package version is set to 1.0.0 for the first deployment, and it should have been 0.1.0 so it is more in synch with the contracts.